src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier_transports.php

```diff
@@ -133,5 +132,11 @@
             ->parent('notifier.transport_factory.abstract')
             ->tag('chatter.transport_factory')
             ->tag('texter.transport_factory')
+
+            ->set('notifier.transport_factory.sms77', Bridge\Sms77\Sms77TransportFactory::class)
```

Comment:
I don't think we need this line.
Here is a PR that deprecates a bridge, we should take inspiration from it I think: #50296